syncano/models/fields.py

```diff
@@ -276,7 +276,7 @@ def __init__(self, *args, **kwargs):
 
     def validate(self, value, model_instance):
         super(ChoiceField, self).validate(value, model_instance)
-        if self.choices and value not in self.allowed_values:
+        if self.choices and value and value not in self.allowed_values:
```

Comment:
I think you should check for `value is None` explicitly although it would only matter once we get a choicefield that has 0, False, '' as a valid choice ;)